gtp2ogs.js

```diff
@@ -430,6 +430,22 @@ if (argv.id) {
     console.log("Warning: --id alias is no longer supported. Use --username instead.");
 }
 
+if (argv.maxmaintime && (argv.maxmaintimeranked || argv.maxmaintimeunranked)) {
+    console.log("Warning: You are using --maxmaintime in combination with --maxmaintimeranked or/and --maxmaintimeunranked. \n Use either --maxmaintime alone , OR --maxmaintimeranked with --maxmaintimeunranked. \n But don't use the 3 --maxmaintime arguments at the same time. ");
+}
+
+if (argv.minmaintime && (argv.minmaintimeranked || argv.minmaintimeunranked)) {
+    console.log("Warning: You are using --minmaintime in combination with --minmaintimeranked or/and --minmaintimeunranked. \n Use either --minmaintime alone , OR --minmaintimeranked with --minmaintimeunranked. \n But don't use the 3 --minmaintime arguments at the same time. ");
```

Comment:
it should be "and/or" not "or/and".

Reply:
i thought of this first, but one person is more likely to use 2 rather than 3 "family" arguments

Reply:
I'm just commenting on the grammar. In english you write "and/or". You never write "or/and".

Reply:
Yeah it should be and/or